src/schema/v2/artwork/index.ts

```diff
@@ -970,10 +973,14 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
       isInShow: {
         type: GraphQLBoolean,
         description: "Is this artwork part of a current show",
-        resolve: ({ id }, _options, { relatedShowsLoader }) =>
-          relatedShowsLoader({ active: true, size: 1, artwork: [id] }).then(
-            ({ body: shows }) => shows.length > 0
-          ),
+        resolve: ({ id, show_ids }, _options, { showsLoader }) =>
+          show_ids && show_ids.length > 0
+            ? showsLoader({
+                status: EventStatusEnums.getValue("ACTIVE")?.value,
```

Comment:
I don't think you need to add an enum (which is for a client to use) - this is a query param to a Gravity API call, so it could just be `status: "active"` - less indirection.

Reply:
Will do. I figured since we have a constant matching the values allowed by the upstream API, I should use it, but this is even simpler.